tools/pythonpkg/src/pyresult.cpp

```diff
@@ -326,6 +326,7 @@ py::str GetTypeToPython(const LogicalType &type) {
 	}
 	case LogicalTypeId::JSON:
 	case LogicalTypeId::VARCHAR:
+	case LogicalTypeId::UUID:
 		return py::str("STRING");
```

Comment:
Ideally this should be the Python type that this duckdb type converts to, which for UUID is not a string, but a `uuid.UUID` so can we change this to "UUID" for this case instead?

Reply:
sure thing!